### PR TITLE
Update ovirt and ovirt_node to collect additional information (RHV PKI) (RHBZ1845877)

### PR DESCRIPTION
**Pull request by lveyde**
---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [ X] Is the commit message split over multiple lines and hard-wrapped at 79 characters?
- [ X] Is the subject and message clear and concise?
- [ X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [ X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [X ] If this commit closes an existing issue, is the line `Closes: #ISSUENUMBER` included in an independent line?
- [ X] If this commit resolves an existing pull request, is the line `Resolves: #PRNUMBER` included in an independent line?

### CODE REVIEW

**Comment by dougsland** [comment]
LGTM

File: sos/report/plugins/ovirt.py
Comment (TurboTurtle) [inline sos/report/plugins/ovirt.py:118]: Since the vast majority of the added commands are the same, just targeting different certs, I'd suggest something like this:

```python
certs = [
    '/etc/pki/ovirt-engine/ca.pem',
    '/etc/pki/ovirt-engine/apache-ca.pem'
# etc...
]
self.add_cmd_output([
    "openssl x509 -in %s -text -noout" % c for c in certs
])
```

File: sos/report/plugins/ovirt.py
Comment (TurboTurtle) [inline sos/report/plugins/ovirt.py:124]: Similar thing here. Iterate over a list of certs, which makes maintenance easier in the long run.

File: sos/report/plugins/ovirt_node.py
Comment (TurboTurtle) [inline sos/report/plugins/ovirt_node.py:54]: Same as above - iterate over these using a list.

**Review by TurboTurtle — CHANGES_REQUESTED** [review]
The LGTM warnings needs to be resolved before merging. The content being collected seems fine, just need to refactor the approach.

**Review by TurboTurtle — APPROVED** [review]

**Comment by dougsland** [comment]
@TurboTurtle hello there, anything else from our side to complete the merge? Thanks!

**Comment by TurboTurtle** [comment]
We use a 2 maintainer ack process in sos.

So @pmoravec or @bmr-cymru or @BryanQuigley would need to ack as well.

File: sos/report/plugins/ovirt.py
Comment (bmr-cymru) [inline sos/report/plugins/ovirt.py:100]: No whitespace inside lists please.

File: sos/report/plugins/ovirt.py
Comment (bmr-cymru) [inline sos/report/plugins/ovirt.py:118]: No need to wrap the command in a list here - `add_cmd_output()` accepts either a list of strings or a single string as the command argument.

**Review by bmr-cymru — CHANGES_REQUESTED** [review]
One minor formatting glitch but other than that looks good from the sos side.

**Review by bmr-cymru — APPROVED** [review]

**Comment by bmr-cymru** [comment]
I can't really comment on the ovirt changes although they all appear logical and in keeping with the rest of the plugin - there's a one bit of extra whitespace in one patch but other than that looks good.

**Pull request closed without merging**
